Article/Article.js

```diff
@@ -3,24 +3,26 @@
 class Article {
   constructor(element) {
     // assign this.element to the passed in article element
-    this.element;
+    this.element = element;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.element.querySelector('.expandButton')
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerHTML = 'expand';
     // Set a click handler on the expandButton reference (or article element), calling the expandArticle method.
-    
+    this.element.addEventListener('click', () => this.expandArticle());
   }
 
   expandArticle() {
```

Comment:
Project looks great but you could have done an if statement here to have the innertext say expand or close depending on if the article is opened or not.